webpack/ForemanTasks/Components/TasksDashboard/Components/StoppedTasksCard/StoppedTasksCard.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card } from 'patternfly-react';
+import classNames from 'classnames';
+import { translate as __ } from 'foremanReact/common/I18n';
+import './StoppedTasksCard.scss';
+import {
+  getTableItemClassName,
+  shouleBeSelected,
+} from './StoppedTasksCardHelpers';
+
+const StoppedTasksCard = ({
+  className,
+  error,
+  warning,
+  success,
+  timePeriod,
+  focusedOn,
+  onTitleClick,
+}) => {
+  const data = { error, warning, success };
+
+  return (
+    <Card
+      className={classNames(
+        'tasks-donut-card',
+        'stopped-tasks-card',
+        className,
+        {
+          'selected-tasks-card': shouleBeSelected(focusedOn),
+        }
+      )}
+    >
+      <Card.Title onClick={onTitleClick}>{__('Stopped')}</Card.Title>
+      <Card.Body>
+        <table className="table table-titles">
+          <tbody>
+            <tr>
+              <td />
+              <td>Total</td>
+              <td>{timePeriod}</td>
+            </tr>
+          </tbody>
+        </table>
+        <table className="table table-bordered table-striped">
+          <tbody>
+            {Object.keys(data).map((item, index) => (
```

Comment:
`Object.entries` makes more sense here since you need the `key` and the `value`.
The `index` can be replaced by the `key`

```js
Object.entries(data).map(([key, col]) => ... )
```

Reply:
Changed, thanks!